src/components/common.js

```diff
@@ -27,6 +27,22 @@ export function validateProps(props : Object, required : boolean = true) {
         }
     }
 
+    if (props.style && props.style.label) {
+        if (props.style.label !== 'checkout' || props.style.label !== 'credit') {
```

Comment:
I think this check will always be true? Guessing it should be `&&` rather than `||`.